ios/node.rb

```diff
@@ -0,0 +1,30 @@
+require('json')
+require_relative('pod_helpers')
+
+def find_node
+  node_bin = `which node`.strip!
+  return node_bin if `file #{node_bin}`.include? 'Mach-O'
+
+  # If a wrapper is found, remove the path from `PATH` and try again. This can
+  # sometimes happen when `pod install` is run inside a "virtual" environment
+  # like Turborepo.
```

Comment:
Can we add a link here? Turboreoo isn't obvious to me what this is, and I might think it's related to turbo modules.